packages/cozy-client/src/associations/HasOne.js

```diff
@@ -0,0 +1,17 @@
+import get from 'lodash/get'
+import Association from './Association'
+
+export default class HasOne extends Association {
+  get raw() {
+    return this.target[this.name]
+  }
+
+  get data() {
+    return this.get(this.doctype, this.raw)
+  }
+
+  static query(doc, client, assoc) {
+    const relationship = get(doc, `relationships.${assoc.name}.data`, [])[0]
```

Comment:
Si c'est une HasOne, pourquoi il y a une liste ? On pourrait utiliser `const relationship = this.raw` ?